src/components/data/datagrid/datagrid.stories.tsx

```diff
@@ -416,17 +416,17 @@ export const DateRangeFilter: Story = {
         type: "string",
       },
       {
-        filterTransform: (data) => {
-          const { dateOfBirth, _data } = data;
-          const [dateOfBirth__gte = "", dateOfBirth__lte = ""] =
-            String(dateOfBirth).split("/");
-          return { dateOfBirth__gte, dateOfBirth__lte, ..._data };
-        },
         name: "dateOfBirth",
         type: "daterange",
       },
     ],
     filterable: true,
+    filterTransform: (data) => {
```

Comment:
I am wondering: if I understand correctly, now we will have one function that has to handle the filters transform for all the fields. If that's the case, it could be harder to maintain than specifying the filters transform on the fields themselves :thinking: 

Maybe the filter transforms could still be specified per field and then when they need to be applied we could iterate over the fields that are active and pass to a filter the output of the previous one?

Also, how does this behave in the case of selectable columns when a particular field is disabled?

Reply:
> I am wondering: if I understand correctly, now we will have one function that has to handle the filters transform for all the fields. If that's the case, it could be harder to maintain than specifying the filters transform on the fields themselves 🤔
 
 Yes, as I there might be (rare) cases where another filter's value needs to be used as well. 
 
 > Maybe the filter transforms could still be specified per field and then when they need to be applied we could iterate over the fields that are active and pass to a filter the output of the previous one?
 
 Wouldn't this mean that the order the fields is affect the outcome? The order which is used for presentational purposes (and in the future maybe dynamic?
 
 > Also, how does this behave in the case of selectable columns when a particular field is disabled?
 
 I'm not sure what you mean here but I think this is up to the user.